integration_tests/kafka_tests/run.sh

```diff
@@ -18,19 +33,17 @@ source .env
 docker-compose up -d || exit 1
 
 while ! is_kafka_ready; do
-    echo "waiting for Kafka to start and create the topic.."
+    printf "waiting for Kafka to start and create the topics..\n"
```

Comment:
just curious: why? :)

Reply:
From what I heard it's better practice to use `printf` instead of `echo` -- https://unix.stackexchange.com/questions/65803/why-is-printf-better-than-echo